.github/workflows/ci.yaml

```diff
@@ -77,6 +78,13 @@ jobs:
           BUILD_DATE=$(TZ='US/Los_Angeles' date '+%Y-%m-%d')
           echo "BUILD_DATE=${BUILD_DATE}" >> $GITHUB_OUTPUT
 
+      - name: Set base image
+        id: base
+        shell: bash -x -e {0}
+        run: |
+          IMG=${{ inputs.CUDA_IMAGE }}
+          echo "BASE_IMAGE=${IMG:-'NOT SPECIFIED'}" >> $GITHUB_OUTPUT
```

Comment:
If we already made `inputs.CUDA_IMAGE` default to 'NOT SPECIFIED', then we don't need to assign defaults again here.